content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -21,7 +21,7 @@ We present here a 1D modelisation of the Magnetotelluric waves in a layered Eart
 
 .. _binder: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
 
-Magnetotelluric is a widely used method, especially for imaging geothermal. Its ability to image deep structure, up to several kilometers depth, is unique in EM geophysics. It is a passive method that use plane waves generated mostly in the Earth's Atmosphere. High frequency waves are mainly produced by lightning strikes all around the globe, traveling through the Earth's Ionosphere that acts as a waveguide. Low frequency waves are produced through the interaction of the Earth's Ionosphere with solar wind and Earth's magnetic field.
+Magnetotelluric (MT) is a widely used method, especially for imaging geothermal. Its ability to image deep structure, up to several kilometers depth, is unique in EM geophysics. It is a passive method that use plane waves generated mostly in the Earth's Atmosphere. High frequency waves are mainly produced by lightning strikes all around the globe, traveling through the Earth's Ionosphere that acts as a waveguide. Low frequency waves are produced through the interaction of the Earth's Ionosphere with solar wind and Earth's magnetic field.
```

Comment:
'Magnetotellurics (MT)' or 'The Magnetotelluric method'  

(just a tip, not required - if you split the lines CMD+ALT+Q in sublime, it makes versioning a bit more transparent. then each line is versioned independently)